### ISSUE TITLE
Initialize linear weights before softmax in MLM

### ISSUE THREAD

**Issue by vkhangpham** [facebookresearch/fairseq#4140]
#### What is your question?
Hi guys, I am training a masked language model using **masked_lm task** and **masked_lm criterion**. Now I want to initialize the linear weights (before softmax) of the model, then how could I do it? It seems that there is no command-line argument to achieve the thing. I really appreciate any reply from you guys.

#### Code

!CUDA_VISIBLE_DEVICES=0 fairseq-train \
    --task masked_lm data-bin/mlm \
    --save-dir checkpoints/mlm \
    --restore-file checkpoints/mlm/checkpoint_last.pt \
    --max-epoch 20 --save-interval 1 --no-epoch-checkpoints \
    --arch cl_encoder --criterion masked_lm \
    --optimizer adam --lr 0.0001 --stop-min-lr 1e-09 \
    --dropout 0.1 \
    --batch-size 128 --tokens-per-sample 256 \
    --ddp-backend=legacy_ddp

#### What's your environment?

 - fairseq Version (e.g., 1.0 or main): main
 - PyTorch Version (e.g., 1.0): 1.10.0+cu111
 - OS (e.g., Linux): Linux (Colab)
 - How you installed fairseq (`pip`, source): pip
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version: 
 - GPU models and configuration:
